src/app/Signup/page.module.css

```diff
@@ -0,0 +1,116 @@
+.SignUpPageContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+}
+
+.EmailContainer {
+  align-items: center;
+  margin-bottom: 1rem;
+}
+.PasswordContainer {
+  align-items: center;
+  margin-bottom: 1rem;
+}
+.ConfirmPasswordContainer {
+  align-items: center;
+  margin-bottom: 2rem;
+}
+.TextBoxLabel {
+  margin-bottom: 0.2rem;
+  display: flex;
+}
+.span {
+  display: flex;
+  justify-content: center;
+}
+
+.CreateAccount {
+  margin: 3rem;
+  font-size: 3rem;
+  margin-bottom: 3rem;
+  margin-top: 2rem;
+}
```

Comment:
define `text-align: center;` and delete all occurances of `text-align: center;`  within the `.CreateAccount` classes inside the media queries below